.github/workflows/deploy_prod.yml

```diff
@@ -168,5 +168,5 @@ jobs:
           identifier: azion.azion
           installers-regex: '\.exe$'  # Only `.exe` files
           token: ${{ secrets.RELEASE_GITHUB_TOKEN }}
-      
-      
\ No newline at end of file
+
+          
```

Comment:
Those spaces. 🤔

Reply:
I deleted every space and still looks like this for some reason 🤷